apps/www/_blog/2022-12-14-mfa-auth-via-rls.mdx

```diff
@@ -0,0 +1,149 @@
+---
+ title: 'Multi-factor Authentication via Row Level Security Enforcement'
+ description: MFA Auth with enforcement via RLS
+ author: joel_lee,kangmingtay,stojan
+ image: lw6-hackathon/auth.jpg
+ thumb: lw6-hackathon/auth.jpg
+ tags:
+   - launch-week
+   - auth
+ date: '2022-12-13'
+ toc_depth: 3
+ ---
+
+
+# Multi-factor Authentication via Row Level Security Enforcement
+
+
+## Background
+
+Today, we’re releasing Multi Factor Authentication for everyone. We are also excited to commence the dogfooding of SAML support via the introduction of Single Sign On (SSO) on our dashboard - you can contact us at [growth@supabase.com](mailto:growth@supabase.com) if you want to enable this in your Enterprise or Pro plan.
```

Comment:
Link to `https://docs.supabase.com/guides/platform/sso/enable-sso` here where appropriate.